### DISCUSSION TITLE
Is the L20 Ultra not supported?

### DISCUSSION THREAD

**Discussion by Akkulader** [Tasshack/dreame-vacuum#455] [General]
Hey hey,
it is not in the list of supported devices and when I connect mine to HA, most of the options are grayed out and not usable.
Ive got version v2.0.0b11 installed

Its looking like this:
![dreame](https://github.com/Tasshack/dreame-vacuum/assets/35584697/cbf49d1e-0275-4f23-b1a4-60d4470c6279)

Does anyone know? I would really appreciate some help!
Best regards

**Comment by Tasshack** [comment]
You should install the latest Beta.

**Reply by Akkulader** [reply]
Hey thank you! Now Im on v2.0.0b12 - still every option is grayed out - also tried multiple restarts

**Reply by Tasshack** [reply]
You should checkout the logs for any possible warnings or errors.

**Reply by Akkulader** [reply]
if I try to start it for a room cleaning Im getting this error: homeassistant.exceptions.HomeAssistantError: Unable to discover the device over cloud

BUT Im able to start the cleaning of the whole hose - its okay... i dont get it
![HA](https://github.com/Tasshack/dreame-vacuum/assets/35584697/c9d115d5-c8d1-4921-88a5-7a7518623831)

**Reply by Akkulader** [reply]
Ive got the solution - i was stupid - sorry! and THANK YOU <3
